clang/include/clang/Driver/CommonArgs.h

```diff
@@ -294,6 +294,10 @@ std::string complexRangeKindToStr(LangOptions::ComplexRangeKind Range);
 // Render a frontend option corresponding to ComplexRangeKind.
 std::string renderComplexRangeOption(LangOptions::ComplexRangeKind Range);
 
+// Adds the default clang include path.
+void addDefaultIncludeDir(const ToolChain &TC, const llvm::opt::ArgList &Args,
```

Comment:
Is this declaration needed? It does not seem to be used anywhere else in the patch.